### PR TITLE
Census block groups should highlight when selected

### PR DESCRIPTION
**Pull request by NatHillardUSDS**
Addresses #316 - 
* Adds 5px blue border on selection of feature
* Adds tests and test helpers

Left out for now:
* zooming to block group - there were some bugs with the default `fitBounds` implementation, especially alongside the inability to zoom past 11

### CODE REVIEW

File: client/cypress/support/commands.js [hunk 1]
```diff
@@ -21,3 +21,8 @@ Cypress.Commands.add('getMapCanvas', () => {
   return cy.get('.maplibregl-canvas');
 });
 
+Cypress.Commands.add('waitForMapIdle', (map) => {
+  return new Cypress.Promise((resolve) => {
```
Comment (TomNUSDS) [inline client/cypress/support/commands.js:25]: It seems odd to return a Promise here?
Maybe it should be "wrapped".
` return cy.wrap(new Cypress.Promise((resolve) ...`

I'm not super familiar with Cypress.

If it compiles and works, probably not super important.

**Review by TomNUSDS — DISMISSED** [review]
lgtm

File: client/cypress/support/commands.js (hunk 1 above)
Comment (NatHillardUSDS) [reply client/cypress/support/commands.js:25]: Cypress requires Promise-like objects as returns, as it waits for the results to be available - I believe `wrap` could be used by its parent if we needed it, but at this level we need to indicate that its async

**Review by TomNUSDS — APPROVED** [review]
Reapproved after merge

File: client/cypress/support/commands.js (hunk 1 above)
Comment (TomNUSDS) [reply client/cypress/support/commands.js:25]: I missed replying to this item, but it's more a curiosity versus and an actual issue. I was just expecting something like this if a promise was returned (note the `async`).

`Cypress.Commands.add('waitForMapIdle', async (map) => {`

**Pull request merged**